applications/nrf5340_audio/src/drivers/Kconfig

```diff
@@ -6,32 +6,27 @@
 
 menu "Drivers"
 
-#----------------------------------------------------------------------------#
-menu "Thread priorities"
+menuconfig NRF5340_AUDIO_CS47L63_DRIVER
```

Comment:
Can we somehow combine CONFIG_NRF5340_AUDIO_CS47L63_DRIVER and CONFIG_HW_CODEC_CIRRUS_LOGIC/CONFIG_HW_CODEC_CS47L63 ?